internal/repository/repository.go

```diff
@@ -0,0 +1,18 @@
+package repository
+
+import (
+	"github.com/newUser1337/task-news/internal/repository/article"
+	"github.com/newUser1337/task-news/internal/repository/mgdb"
+)
+
+type Repository struct {
+	Article *article.ArticleRepository
+}
+
+func NewRepository(clientDb any) *Repository {
+	dbConnector := mgdb.NewMongoClient(clientDb)
```

Comment:
why do you create mongo client in a repo instead of passing it as an argument to this function?

Reply:
fixed